### ISSUE TITLE
Unable to upgrade to bun 1.1.2 on Windows

### ISSUE THREAD

**Issue by xameryn** [oven-sh/bun#10010]
### What version of Bun is running?

1.1.1+ca1dbb4eb

### What platform is your computer?

Microsoft Windows NT 10.0.22631.0 x64

### What steps can reproduce the bug?

- run bun upgrade or bun upgrade --canary (before 1.1.2 became an official release)
- get error

### What is the expected behavior?

Bun would upgrade to 1.1.2

### What do you see instead?

```
Bun v1.1.2 is out! You're on 1.1.1

uh-oh: attempt to unwrap error
bun will crash now 😭😭😭

----- bun meta -----
Bun v1.1.1 (ca1dbb4e) Windows x64
UpgradeCommand:
Elapsed: 4481ms | User: 0ms | Sys: 0ms
RSS: 77.52MB | Peak: 77.52MB | Commit: 0.14GB | Faults: 19122
----- bun meta -----

0   00007FF69FF7ED58
1   ???
2   ???
3   ???
4   ???
5   ???
6   ???
7   ???
8   ???
9   ???

Search GitHub issues https://bun.sh/issues or join in #windows channel in https://bun.sh/discord

thread 34408 panic: attempt to unwrap error
???:?:?: 0x7ff69ebbbdd1 in ??? (bun.exe)
???:?:?: 0x7ff69ebb9945 in ??? (bun.exe)
???:?:?: 0x7ff69ebbc406 in ??? (bun.exe)
???:?:?: 0x7ff69ebc90e7 in ??? (bun.exe)
???:?:?: 0x7ff69e8f1603 in ??? (bun.exe)
???:?:?: 0x7ff6a1473ec3 in ??? (bun.exe)
???:?:?: 0x7ffe6674257c in ??? (KERNEL32.DLL)
???:?:?: 0x7ffe6704aa57 in ??? (ntdll.dll)
```

**Comment by xameryn** [comment]
This seems to be the same issue as well
https://github.com/oven-sh/bun/issues/9943#issuecomment-2040837177

**Comment by paperclover** [comment]
i think this issue was fixed in 1.1.2, can you use the powershell script to reinstall, and then if this continue to happen we can confirm it is not fixed.

**Comment by xameryn** [comment]
Can confirm fixed after a clean install 🫡

**Comment by Rohit19060** [comment]
![image](https://github.com/oven-sh/bun/assets/39453065/5ff6c2cc-2bb6-4b47-a9dc-6a5580afcbed)
Still there

**Comment by kizmo** [comment]
i have same issue (`cmd`, `vs code` terminal and `power shell`)
when do `bun upgrade` from `1.1.1` to `1.1.2`

haven't tried fresh install, just upgrade.

` Windows 10 - Build 18363 `



```
Bun v1.1.2 is out! You're on 1.1.1

uh-oh: attempt to unwrap error
bun will crash now 😭😭😭

----- bun meta -----
Bun v1.1.1 (ca1dbb4e) Windows x64 (baseline)
UpgradeCommand:
Elapsed: 7591ms | User: 296ms | Sys: 171ms
RSS: 78.34MB | Peak: 78.35MB | Commit: 133.11MB | Faults: 19543
----- bun meta -----

0   00007FF68393CE58
1   ???
2   ???
3   ???
4   ???
5   ???
6   ???
7   ???
8   ???
9   ???

Search GitHub issues https://bun.sh/issues or join in #windows channel in https://bun.sh/discord

thread 8112 panic: attempt to unwrap error
???:?:?: 0x7ff68257d0f1 in ??? (bun.exe)
???:?:?: 0x7ff68257ac5a in ??? (bun.exe)
???:?:?: 0x7ff68257d736 in ??? (bun.exe)
???:?:?: 0x7ff682589f1f in ??? (bun.exe)
???:?:?: 0x7ff6822c15fe in ??? (bun.exe)
???:?:?: 0x7ff684e2cba3 in ??? (bun.exe)
???:?:?: 0x7ff955137bd3 in ??? (KERNEL32.DLL)
???:?:?: 0x7ff95622ced0 in ??? (ntdll.dll)
```

**Comment by axefrog** [comment]
Confirming failure here too. Just tried to upgrade from 1.1.1 to 1.1.2 a couple of minutes ago.

**Comment by lanoow** [comment]
Still not fixed in v1.1.3, trying to upgrade to v1.1.4 via `bun upgrade` but no solution found.
